.circleci/config.yml

```diff
@@ -152,6 +152,65 @@ jobs:
           name: Build
           command: yarn run build
 
+  integration_tests:
+    docker:
+      - image: circleci/python:3.7.3-node-browsers
+    
+        environment:
+          REACT_APP_INSECURE_AUTH_ALLOWED: true
+          MOZ_HEADLESS: 1
+
+    steps:
+      - checkout
+
+      - setup_remote_docker
+
+      - restore_cache:
+          keys:
+          - v2-dependencies-{{ checksum "yarn.lock" }}
+          # fallback to using the latest cache if no exact match is found
+          - v2-dependencies-
+          - integration-test-dependencies-{{ checksum "tests/integration/requirements.txt" }}
+
+      - run:
+          name: Install node dependencies
+          command: yarn install
+
+      - save_cache:
+          paths:
+            - "node_modules"
+          key: v2-dependencies-{{ checksum "yarn.lock" }}
+
+      - run:
+          name: Install Tox
+          command: sudo pip install tox
```

Comment:
is there a reason why `tox` is not included in the requirements file?

Reply:
The requirements file gets run within a tox step, so to avoid installing everything twice, I install tox by itself before.